test_framework/terraform/aws/sle-micro/user-data-scripts/provision_k3s_agent.sh.tpl

```diff
@@ -18,5 +18,6 @@ elif [ -b "/dev/xvdh" ]; then
   sudo mount /dev/xvdh /var/lib/longhorn
 fi
 
+sudo su -
```

Comment:
Did you test it by running the terraform scripts? 

From my understanding, `su -` will open a new shell and it's not what we  expected here.

Reply:
I see. Roger's comment did mention that.

Reply:
@roger-ryao Let's check this first before merging.

Reply:
Hi @yangchiu & @innobead.
     I would try to test it by running the terraform scripts.

Reply:
Hi @yangchiu:
      Thank you for your reminder. My previous commit did not work as expected. I have made modifications to it and tested it manually as well as by running the Terraform scripts.
    
    
![Screenshot_20230610_182646](https://github.com/longhorn/longhorn-tests/assets/104337648/07db3318-308d-4784-9f60-7d5ce6af61f4)